### ISSUE TITLE
transcriptional program feature gene analysis

### ISSUE THREAD

**Issue by yuzu1999** [wu-yc/neutrophil#1]
Thank you very much for sharing the code. I would like to know how All_obj_time1. rda was obtained, including how to perform cell quality control, dimensionality reduction, clustering, and transcriptional program feature gene analysis. Could you please share this part of the code? Thank you very much!

**Comment by fengshuoliu** [comment]
Agreed, without the detailed Seurat object or processing steps, it's hard to reproduce the results or build on the data.
